deploy/scripts/src/ipv-core/test.ts

```diff
@@ -122,20 +122,20 @@ export function passport (): void {
       'verify page content': (r) => (r.body as string).includes('Enter userId manually')
     }))
 
-  sleep(Math.random() * 2)
+  sleep(Math.random() * 1)
 
   res = group('B01_Passport_02_GoToFullJourneyRoute GET', () =>
     timeRequest(() => {
       const response = http.get(env.orchStubEndPoint + '/authorize?journeyType=full&userIdText=', { tags: { name: 'B01_Passport_02_GoToFullJourneyRoute' } })
-      if (env.staticResources) getStaticResources(response)
+      // if (env.staticResources) getStaticResources(response)
       return response
     },
     {
       'is status 200': (r) => r.status === 200,
       'verify page content': (r) => (r.body as string).includes('Tell us if you have one of the following types of photo ID')
     }))
 
-  sleep(Math.random() * 2)
+  sleep(Math.random() * 1)
```

Comment:
```suggestion
  sleep(Math.random())
```
Well we can drop all these multiplications by `1`

Should rebase off the changes of #330 anyway

Reply:
Ah didn't pay attention that I was reducing it to 1. Apologies.
Have now rebased this branch so all updated.